uniform.go

```diff
@@ -12,12 +12,16 @@ type Uniform struct {
 }
 
 // Jitter the duration by drawing from a uniform distribution
+// over the range [Min, d). Returns Min if d <= Min
 func (u Uniform) Jitter(d time.Duration) time.Duration {
 	drawUniform := rand.Int63n
 	if u.Source != nil {
 		drawUniform = u.Source.Int63n
 	}
 
-	d = time.Duration(drawUniform(int64(d)))
-	return max(d, u.Min)
+	delta := d - u.Min
+	if delta <= 0 {
```

Comment:
It seems like setting `d <= u.Min` is always a mistake, and that it is therefore preferable to make this mistake visible to the user instead of correcting it in an *ad hoc* way.  Let's make this panic instead.

Reply:
Agreed. Fix made